src/e2e/features/createEventWithAllFields.feature

```diff
@@ -0,0 +1,28 @@
+@createEventWithAllFields
+Feature: Event Creation
+
+  Scenario: Verify ECC dashboard page content
+    Given I am on the ECC dashboard page in signed-out state
+    Then I should see the All Events label on the page
+    Then I should see the Search box on the page
+    Then I should see the table headers on the page
+    Then I should see pagination container
+    Then I should see the footer section on the page
+
+    Scenario Outline: Validate and create events with all fields
+    Given I am on the ECC dashboard page in signed-in state
+    And I should see the Create new event button on the page
+    Then I should be able to click on Create new event button
+    Then I am in the create event flow and Basic info page
+    Then I fill out create event Basic info page with <EventDataWithAllFields> and click Next step
+    Then I fill out create event Speakers & Hosts page with <EventDataWithAllFields> and click Next step
+    Then I fill out create event Additional content page with <EventDataWithAllFields> and click Next step
+    Then I fill out create event RSVP page with <EventDataWithAllFields> and click Publish event
+    Then I should check that event is created
+    Then I should be able to search & validate the event on ECC dashboard
```

Comment:
should search and delete be another scenario?

Reply:
There are some challenges to this. We can discuss this offline.

Reply:
Separated Search & delete scenarios